RFC initial design (#1)
iver-wharf/rfcs: README.md

# Wharf RFCs (Request For Comments)

For any substantial change that needs to be thought through in Wharf, we try to
use RFCs for them instead of internally trying to talk it out in meetings.

> Not every descision needs an RFC, nor is any RFC excessive. Altough this is the

Reviewer: ```suggestion
Not every decision needs an RFC, nor is any RFC excessive. This is the
```

start of our journey with RFCs, so we're trying to limit them to changes with
major impact.

Read more about the Wharf RFC process over at the GitHub pages:
<https://iver-wharf.github.io/rfcs/>

## Disclaimer

The documents and markdown files in this repository are documentation of our

> descision making. Only to have a place to look back at, similar to meeting

Reviewer: ```suggestion
decision making. Only to have a place to look back at, similar to meeting
```

> notes or recordings, so we can see our previous descisions.

Reviewer: ```suggestion
notes or recordings, so we can see our previous decisions.
```

**It is not** documentation of how things are meant to be implemented. Do not
take this repository as the source of truth of how Wharf works. Please check
out our [Wharf documentation](https://iver-wharf.github.io/) or the
[source code](https://github.com/iver-wharf/) itself to gain the actual source
of truth.

## Running locally

This repository is hosted using GitHub Pages, but you can also host it locally

> if you so want.

Reviewer: ```suggestion
if you want.
```
